### ISSUE TITLE
[filter-effects] Accessibility Considerations Statement

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/fxtf-drafts#330]
The [APA Working Group suggests an Accessibility Considerations section](https://lists.w3.org/Archives/Public/public-apa-admin/2018Dec/0008.html) for [Filter Effects Module Level 1](https://www.w3.org/TR/filter-effects-1/). Suggested content for the section:

In addition to standard Web Content Accessibility Guidelines 2.1 requirements for images, content authors using filter effects should be mindful of how the filters modify the colors, contrast, size, and animation of images.  The following Level A and AA success criteria could be affected:

* [Success Criterion 1.4.1 Use of Color](https://www.w3.org/TR/WCAG21/#use-of-color). Given a button shaped as an arrow with the text, a filter could modify the background color so that the text is no longer visible. 
* [Success Criterion 1.4.3 Contrast (minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum). Given the example above, a filter could modify the background or text color(s) so  color contrast requirement is not longer met.  
* [Success Criterion 1.4.11 - Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast). Given the example above, a filter could modify the colors so that the contrast between the text and button color, and/or the button color and background no longer meet color contrast requirements. It is particularly important to be mindful of color contrast  requirements when a transparency is present because automated measurement is not available to quantitatively and correctly judge whether sufficient contrast is being provided, thus requiring application of informed human judgement on a case by case basis.
* [Success Criterion 2.3.1 Three Flashes or Below Threshold](https://www.w3.org/TR/WCAG21/#three-flashes-or-below-threshold). Beware of any filters that add animation that may add or change the flash rate.
